deploy/cdk/bin/base.ts

```diff
@@ -103,13 +103,13 @@ new imageProcessing.DeploymentPipelineStack(app, `${namespace}-image-processing-
 const elasticsearchContext = getContextByNamespace('elasticsearch')
 const elasticsearchProps = {
   env,
+  contextEnvName: envName,
   namespace,
   foundationStack,
   ...elasticsearchContext,
 }
 new elasticsearch.ElasticStack(app, `${namespace}-elastic`, elasticsearchProps)
 new elasticsearch.DeploymentPipelineStack(app, `${namespace}-elastic-deployment`, {
-  contextEnvName: envName,
```

Comment:
Deployment props include `...elasticSearchProps` so this is just removed because it's redundant.